docs/user-guide/src/bmo/reboot_annotation.md

```diff
@@ -0,0 +1,69 @@
+# Reboot annotation
```

Comment:
This in my opinion missing to mention an important point on the scope of this annotation that it can be used: reboot annotation is meant to be used on `provisioned` hosts only, and it is not responsible for managing unprovisioned hosts

Reply:
Good point! Added at the top of the document.